examples/nextjs-form/pages/index.tsx

```diff
@@ -31,13 +33,22 @@ export default function Example() {
   const updateMyPresence = useUpdateMyPresence();
 
   /**
-   * useObject is used to share a simple state to all the users in the room.
-   * It's using the storage block so the data is persisted even after all the users leave the room.
-   * For more information: https://liveblocks.io/docs/api-reference/liveblocks-react#useObject
+   * useStorage is used to read and stay in sync with the shared state, which
+   * all users in the room see. It's using Liveblocks Storage so the data is
+   * persisted even after all the users leave the room. For more information:
+   * https://liveblocks.io/docs/api-reference/liveblocks-react#useStorage
    */
-  const data = useObject("logo");
+  const logo = useStorage((root) => root.logo);
 
-  if (!data) {
+  const updateName = useMutation(({ storage }, name: string) => {
+    storage.get("logo").set("name", name);
+  }, []);
+
+  const updateTheme = useMutation(({ storage }, theme: "light" | "dark") => {
+    storage.get("logo").set("theme", theme);
+  }, []);
+
+  if (!logo) {
```

Comment:
I was also on it yesterday but forgot to communicate it, thanks for doing it! I was wondering if suspense could have been a good call here 🤔

Reply:
Yeah, I considered it, but since this is the only null check in the entire example, I just decided to keep it as-is. I think it's fine.

Reply:
Glad we have a similar reaction, proves 0.18 is straightforward to use!